locallib.php

```diff
@@ -1083,6 +1085,19 @@ function convertsecondstostring($secs) {
     return $hours.$minutes.':'.$seconds;
 }
 
+/**
+ * Gets number of attempts remaining for the user.
+ *
+ * @param int $time timestamp in Unix epoch time
+ * @return string $timestamp timestamp in Europe/Helsinki timezone format dd.mm.yyyy hh.mm:ss
+ */
+function timestampformatter($time) {
+    date_default_timezone_set('Europe/Helsinki');
```

Comment:
I don't know if its good idea to overwrite locales set in Moodle's settings?
![image](https://user-images.githubusercontent.com/6292793/165576597-411100f3-2fd6-4d31-9494-c32e30d42b3f.png)

Reply:
Locally I get the right time only by doing this overwriting even though my timezone settings are right. Why could this be?

Reply:
https://docs.moodle.org/dev/Time_API Finally found documentation that I looked for. Could this be handy? I'm worried about if there is users from Sweden for example, then times are one hour off.

Reply:
The problem here seems to only be with the timecreated/timemodified-confusion that will be fixed in another PR.